src/runtime/pipeline/pipeline_struct.h

```diff
@@ -21,24 +21,74 @@
 #include <assert.h>
 #include <dlpack/dlpack.h>
 #include <dmlc/json.h>
+#include <tvm/runtime/ndarray.h>
+#include <tvm/runtime/packed_func.h>
 
 #include <limits>
 #include <string>
 #include <unordered_map>
+#include <utility>
 #include <vector>
+namespace tvm {
+namespace runtime {
+#define GLOBAL_MODULE_INDEX -1
+/*!
+ *\brief The mapping of a module output and a global output in the graph module.
+ */
+struct GlobalOutputPair {
+  int mod_output_idx;
+  int global_output_idx;
+  GlobalOutputPair(const int idx, const int gidx) : mod_output_idx(idx), global_output_idx(gidx) {}
+  GlobalOutputPair() {}
+};
+
+/*!
+ *\brief Use the module index and the output index to specify a module output.
+ */
+struct ModuleOutputPair {
+  int mod_idx;
+  int output_idx;
+  ModuleOutputPair(const int midx, const int idx) : mod_idx(midx), output_idx(idx) {}
+  ModuleOutputPair() {}
+};
 /*!
  * \brief All binding information of a output interface.
  */
-struct OutputBindings {
+class ConfigBindings {
+ private:
   /*!\brief Output interface binding information, 'int' is the index of the module that
    *  uses this output data as the input interface data, 'string' is the input interface name
    *  of the module.
    */
   std::unordered_map<int, std::string> bindings;
```

Comment:
Private variables need `_` as the suffix.

Reply:
fixed.